src/test/tests/applicabilityScan.test.ts

```diff
@@ -114,125 +127,229 @@ describe('Contextual Analysis Scan Tests', () => {
-                assert.equal(applicableCve.size, test.expectedEvidence.length);
+                it('Check number of applicable Cve in response', () => {
+                    assert.equal(applicableCve.size, test.expectedEvidence.length);
+                });
+
                 for (let evidence of test.expectedEvidence) {
-                    let details: CveApplicableDetails | undefined = applicableCve.get(evidence.cve);
-                    assert.isDefined(details, 'expected evidence for ' + evidence.cve + ' in: ' + Array.from(applicableCve.keys()));
-                    assert.equal(details?.fixReason, evidence.reason);
-                    assert.include(details?.fullDescription, evidence.descriptionPartial);
-                    assert.equal(details?.fileEvidences.length, evidence.files.length);
-                    for (let fileEvidence of evidence.files) {
-                        let fileIssue: FileIssues | undefined = details?.fileEvidences.find(fileIssue => fileIssue.full_path === fileEvidence.file);
-                        assert.isDefined(
-                            fileIssue,
-                            'expected evidence for file ' + fileEvidence.file + ' in: ' + details?.fileEvidences.map(f => f.full_path)
-                        );
-                        assert.equal(fileIssue?.locations.length, fileEvidence.locations.length);
-                        for (let locationEvidence of fileEvidence.locations) {
-                            let location: FileRegion | undefined = fileIssue?.locations.find(
-                                fileLocation =>
-                                    fileLocation.snippet?.text === locationEvidence.text &&
-                                    fileLocation.startLine === locationEvidence.location[0] &&
-                                    fileLocation.endLine === locationEvidence.location[1] &&
-                                    fileLocation.startColumn === locationEvidence.location[2] &&
-                                    fileLocation.endColumn === locationEvidence.location[3]
-                            );
-                            assert.isDefined(
-                                location,
-                                'expected evidence for text ' +
-                                    locationEvidence.text +
-                                    ' at location: Start(' +
-                                    locationEvidence.location[0] +
-                                    ',' +
-                                    locationEvidence.location[2] +
-                                    ') End(' +
-                                    locationEvidence.location[1] +
-                                    ',' +
-                                    locationEvidence.location[3]
-                            );
+                    describe('Check ' + evidence.cve + ' applicable evidences', () => {
```

Comment:
`describe` is used to structure your test suite. There is no reason to use it as an individual test case.